### ISSUE TITLE
object 'extent' not found

### ISSUE THREAD

**Issue by EmilyPhelps** [hiweller/recolorize#6]
Hello,

I've been following your tutorial on _Polistes fuscatus,_ trying to see if it could work for colour patterns in the Corydoras.

I have run the alignLan() and the majority of images have come out great.

However, I cant get the extents from my images. Im new to image analysis (mainly work in 'omics) so a little confused by what this means and how to troubleshoot.

```
extent_list <- lapply(imageList_aligned, extent)
Error in match.fun(FUN) : object 'extent' not found
```

**Comment by EmilyPhelps** [comment]
Okay I fixed it!

FYI discussing [this tutorial ](https://hiweller.rbind.io/post/recolorize-patternize-workflow/) 

I needed to call the library(raster) to get this to work as a function. Then it extracts all the extents. 

I was confused because earlier when I ran alignLan I got a warning saying:
``` 
Warning messages:
1: [rast] unknown extent 
```
But when I explored the object I found the extent section and those were present. 

TL;DR: If you get this error you need to load the raster package. This is not in the tutorial at the time of writing.

**Comment by hiweller** [comment]
Thank you for not only catching this error, but finding your own solution and posting about it! This should now be fixed in the original post (with [this commit](https://github.com/hiweller/hannahiweller/commit/ee6c1a00f4e73d4cb02edd4dc5fd5598fed9c6c3)).

For full closure/the curious:

That error never got caught when I compiled the blog post because that chunk of code doesn't get evaluated. I just cooking-show-style load the list of image arrays to save on processing time. Another solution if you'd rather not load the raster package is to just access the extent function from the raster package using the double colon (`::`) notation:

```{r}
# save raster extents for later conversion:
extent_list <- lapply(imageList_aligned, raster::extent)
```

The `unknown extent` warning message can be safely ignored (since we set the extent explicitly in a different step).
